platform-core-module/src/main/java/com/softicar/platform/core/module/AGCoreModuleInstance.java

```diff
@@ -25,6 +25,11 @@ public static AGCoreModuleInstance getInstance() {
 		return AGCoreModuleInstance.TABLE.get(SINGLETON_INSTANCE_ID);
 	}
 
+	public static String getSystemIdentifier() {
```

Comment:
hmm, why is the method called getSystemIdentifier() but the field is called SYSTEM_NAME? Should not both be equal? Though, I have no preference for either one.

Reply:
its called like that cause I didnt change it from `AGLiveSystemConfiguration` :D

Reply:
hmpf, well